### ISSUE TITLE
terminal transparency still not working

### ISSUE THREAD

**Issue by agaida** [qterminal/qterminal#157]
cant reopen the closed bug - so i open a new one.

**Comment by jleclanche** [comment]
@agaida added you

**Comment by agaida** [comment]
thanks

**Comment by pvanek** [comment]
the transparency requires support in environment - do you use some kind of compositor, please? (compton, kwin,...)

**Comment by agaida** [comment]
compton in one system, xfwm4 in my main systems - i should add that application transparency work fine

**Comment by LeonardKoenig** [comment]
Having this too (I guess). I turned it on but didn't notice any difference, using compton here.

The packaging on Arch is actually quite straight-forward. It just calls

```bash
cmake "$srcdir/$_pkgname" \
		-DCMAKE_INSTALL_PREFIX=/usr \
		-DCMAKE_INSTALL_LIBDIR=lib \
		-DUSE_SYSTEM_QXT=OFF \
		-DUSE_QT5=true
make
```

and then the usual make install. Maybe it doesn't work because of the prefixes as some stuff is hardcoded (shot in the dark)?

For me application transparency is working too.

**Comment by agaida** [comment]
Fixed with  pull request #169 from antis81/antis81/transparent-terminal
